From 5b877edd31024fd0cfcbfad1ba0c69da2ac9b614 Mon Sep 17 00:00:00 2001
From: Maxim Uvarov <maxim.uvarov@linaro.org>
Date: Fri, 28 Aug 2020 22:20:10 +0300
Subject: [PATCH] efi: change 'env -e -i' usage syntax

'env -e -i' syntax was changed from "," to ":". Account for this also
in the documentation.

Fixes: 2b3fbcb59f41 ("efi_loader: use ':' as separator for setenv -i")
Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>
Correct the usage description for setenv -e too.
Reviewed-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/nvedit.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmd/nvedit.c b/cmd/nvedit.c
index 9f145dd284..7fce723800 100644
--- a/cmd/nvedit.c
+++ b/cmd/nvedit.c
@@ -1477,7 +1477,7 @@ static char env_help_text[] =
 	"env select [target] - select environment target\n"
 #endif
 #if defined(CONFIG_CMD_NVEDIT_EFI)
-	"env set -e [-nv][-bs][-rt][-at][-a][-i addr,size][-v] name [arg ...]\n"
+	"env set -e [-nv][-bs][-rt][-at][-a][-i addr:size][-v] name [arg ...]\n"
 	"    - set UEFI variable; unset if '-i' or 'arg' not specified\n"
 #endif
 	"env set [-f] name [arg ...]\n";
@@ -1541,7 +1541,7 @@ U_BOOT_CMD_COMPLETE(
 	"set environment variables",
 #if defined(CONFIG_CMD_NVEDIT_EFI)
 	"-e [-guid guid][-nv][-bs][-rt][-at][-a][-v]\n"
-	"        [-i addr,size name], or [name [value ...]]\n"
+	"        [-i addr:size name], or [name [value ...]]\n"
 	"    - set UEFI variable 'name' to 'value' ...'\n"
 	"      \"-guid\": GUID xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx\n"
 	"      \"-nv\": set non-volatile attribute\n"
-- 
2.28.0

